org.civicrm.reqangular/test/services/api_test.js

```diff
@@ -108,14 +108,9 @@ define([
       });
 
       function expectAndSendPOST(returnValue) {
-        var entity = 'send_post_entity',
-          action = 'send_post_action';
-
-        $httpBackend
-          .whenPOST('/civicrm/ajax/rest')
-          .respond(returnValue);
+        $httpBackend.whenPOST('/civicrm/ajax/rest').respond(returnValue);
 
-        return api.sendPOST(entity, action);
+        return api.sendPOST('send_post_entity', 'send_post_action');
```

Comment:
be consistent, you use these names in expectAndSendGET
```js
var action = 'get', entity = 'DummyEntity';
```
so do the same here. Even better, just use the same variables for both expectAndSendGET and expectAndSendPOST, no?